### ISSUE TITLE
UX of opened tasks from inside the widget

### ISSUE THREAD

**Issue by fbruetting** [dmfs/opentasks#892]
When you tap on a task from the widget and then delete it, OpenTasks currently shows another task. Even tasks of other categories, which are not in the category-scope of the widget, get displayed, what might be problematic.

I’d like to propose to quit OpenTasks instead of showing another task, because directly tapping on a task of the widget is a task-specific action which gets completely finished when that task gets deleted. Showing up other tasks interrupts the user’s workflow, because there is no need or advantage to show other, usually completely unrelated, tasks.

**Comment by dmfs** [comment]
Right, I see the problem. It happens when you leave the details view via the home button and open another task. In that case the details view is not closed and the details view of the new task is stacked on top of the old one. It shouldn't happen when you leave the details view with the back button.
Anyway, we should start a new stack when opening the task from the widget, so the back button always takes you back to the home screen (or wherever you're coming from).

**Comment by fbruetting** [comment]
I also recognized, that sometimes when I tap on the “+” button of the widget, OpenTasks opens up and shows some complete different task – instead of creating a new and empty one.
